llama-index-integrations/embeddings/llama-index-embeddings-oci-genai/pyproject.toml

```diff
@@ -0,0 +1,57 @@
+[build-system]
+build-backend = "poetry.core.masonry.api"
+requires = ["poetry-core"]
+
+[tool.codespell]
+check-filenames = true
+check-hidden = true
+# Feel free to un-skip examples, and experimental, you will just need to
+# work through many typos (--write-changes and --interactive will help)
+skip = "*.csv,*.html,*.json,*.jsonl,*.pdf,*.txt,*.ipynb"
+
+[tool.llamahub]
+contains_example = false
+import_path = "llama_index.embeddings.oci_genai"
+
+[tool.llamahub.class_authors]
+OCIGenAIEmbeddings = "llama-index"
```

Comment:
feel free to use your github username instead of `llama-index` here

Reply:
addressed